submissions/rubyAdeniregun/Database/Queries.sql

```diff
@@ -0,0 +1,44 @@
+/* Question 1: Query to find the total number of passengers that survived 
+The total number of passengers who survived = 342. */
+SELECT COUNT(*)
+FROM accident_records 
+WHERE passenger_survived LIKE 1; 
+
+
+/* Question2: Query to find the total number of passengers who did not survive
+The total number of passengers who did not survive = 549. */
+SELECT COUNT(*)
+FROM accident_records 
+WHERE passenger_survived LIKE 0; 
+
+
+/* Question 3: Query to get the name and sex of passengers under 
+the age of 27 that embarked at Queenstown and Cherbourg? */
+ SELECT registered_passengers.full_name, registered_passengers.sex 
+ FROM registered_passengers LEFT JOIN passengers_records 
+ ON registered_passengers.id = passengers_records.registered_passengers_id
+ WHERE age < 27 AND (point_of_embarkation= 'Q' OR point_of_embarkation = 'C');
+
+  
+/* Question 4: Query to get how many of the passengers that embarked at Southampton survived? 
+Passengers that embarked at Southampton survived = 218 */
+SELECT COUNT(passengers_records.registered_passengers_id)
+FROM passengers_records INNER JOIN accident_records 
+ON passengers_records.registered_passengers_id = accident_records.registered_passengers_id
+WHERE point_of_embarkation LIKE 'S' AND passenger_survived LIKE 1;
```

Comment:
Instead of using `LIKE` use `=`. This is not the best case to use `LIKE`. Kindly read more on the use of `LIKE`